pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -1 +1,111 @@
 package manifests
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	hiveext "github.com/openshift/assisted-service/api/hiveextension/v1beta1"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	agentClusterInstallFilename = filepath.Join(clusterManifestDir, "agent-cluster-install.yml")
```

Comment:
Nit - using "yml" for the extension is consistent with infraenv.yml but different than pull-secret.yaml. Either is fine, just wondering if we should be consistent.